_posts/2020-10-01-intro-developer-advocate.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title:  "Hello I'm Brian, Presto Developer Advocate"
+author: Brian Olsen
+excerpt_separator: <!--more-->
+---
+
+Hello, Presto nation!
+
+I wanted to take a moment to introduce myself! My name is Brian and I’m a new
+developer advocate working at Starburst. In this blog, I'll give you a little
```

Comment:
Let me give you a little background on how I got here, and cover how my role can hopefully help the Presto community.

Reply:
I'm going to remove "hopefully" as that sounds like I'm already doubting myself.